.golangci.yml

```diff
@@ -6,93 +6,34 @@ run:
   go: '1.19'
 
 linters:
-  # please, do not use `enable-all`: it's deprecated and will be removed soon.
-  # inverted configuration with `enable-all` and `disable` is not scalable during updates of golangci-lint
```

Comment:
I agree that we can remove the comment about enable-all being deprecated, but I think the second comment `inverted configuration with `enable-all` and `disable` is not scalable during updates of golangci-lint` is still valid. I think it will be better to use `disable-all` and `enable`, but you can go ahead and enable to the specific linters you want. It will make it easier to review upgrades this way too

Reply:
So, I will revert it for now.

Reply:
I see. Ok, I'll approve this PR right now so that we can get the lint check passing again. And feel free to open a new PR with `enable-all` and `disable` so we can discuss that separately.